lib/samson/console_extensions.rb

```diff
@@ -0,0 +1,19 @@
+module Samson
+  module ConsoleExtensions
+    def login(role = Role::SUPER_ADMIN)
+      user = User.where(role_id: role.id).last!
```

Comment:
When I try to reproduce things I usually don't care what user I am ...
being able to pass specific users sounds like a nice feature though ...
How about user=User.first ... which should be a super admin and then it can
be overwritten

On Wed, Jun 29, 2016 at 12:05 PM, Jon Moter <notifications@github.com>
wrote:

> In lib/samson/console_extensions.rb
> <https://github.com/zendesk/samson/pull/1113#discussion_r69008260>:
>
> > @@ -0,0 +1,19 @@
> > +module Samson
> > +  module ConsoleExtensions
> > +    def login(role = Role::SUPER_ADMIN)
> > +      user = User.where(role_id: role.id).last!
>
> Just seems weird to specify a Role, and wind up as whoever the last person
> with that role is.
>
> I'd be more inclined to do something like login_as "jmoter@zendesk.com",
> or login_as User.find(1234), to set current user as the user of your
> choice.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1113/files/1f9f87078d669161b83dfba2f134fde079d9b8bb#r69008260>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZya3uiO8ATX1K1YJfJskCIj8Bbx5ks5qQsITgaJpZM4JAwrS>
> .
>